zmq.hpp

```diff
@@ -2232,6 +2232,8 @@ class monitor_t
 
     ZMQ_NODISCARD const void *handle() const ZMQ_NOTHROW { return _monitor_socket.handle(); }
 
+    operator socket_ref() ZMQ_NOTHROW { return (zmq::socket_ref) _monitor_socket; }
```

Comment:
Again, I'd not add an operator here but rather a named `socket()` member function. Actually, then the `handle`/`socket_handle` function is not necessary at all, since its handle can be accessed through the `socket_ref`.

Reply:
Agreed.